bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestRackawareEnsemblePlacementPolicy.java

```diff
@@ -1689,6 +1689,84 @@ public void testNewEnsembleWithPickDifferentRack() throws Exception {
         }
     }
 
+    @Test
+    public void testNewEnsemblePickLocalRackBookiesByHostname() throws Exception {
```

Comment:
This test cannot cover new features, whether there are new features or not, the test will pass. I think we should check if the host and rack information of the localNode align with the expectations.